tests/unit/deposit/test_zenodo_api.py

```diff
@@ -156,3 +159,129 @@ def test_simple_rest_flow(api, api_client, db, es, location, users,
         headers=auth_headers,
     )
     assert response.status_code == 403
+
+
+@pytest.mark.parametrize('user_info,status', [
+    # anonymous user
+    (None, 401),
+    # owner
+    (dict(email='info@zenodo.org', password='tester'), 200),
+    # not owner
+    (dict(email='test@zenodo.org', password='tester2'), 403),
+    # admin user
+    (dict(email='admin@zenodo.org', password='admin'), 200),
+])
+def test_read_deposit_users(api, api_client, db, users, deposit, json_headers,
+                            user_info, status):
+    """Test read deposit by users."""
+    deposit_id = deposit['_deposit']['id']
+    with api.test_request_context():
+        with api.test_client() as client:
+            if user_info:
+                # Login as user
+                with client.session_transaction() as sess:
```

Comment:
Can ``login_user_via_view `` in https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/testutils.py#L69 be used instead?